infra/.gitignore

```
@@ -3,4 +3,9 @@
 **connector-sa-key.json
 
 # PEM files used in local cert generation; don't want to commit them
-**.pem
\ No newline at end of file
+**.pem
+
+
+# standard demos directory, so we can create demo examples for customers without mistakenly in a
+# consistent location that won't be inadvertently committed to repo
+examples-demo/**
```

With this, even `README.md` included in this PR will be out of source control once this ignore is merged... not sure if something like the suggestion might work to only include that file:

```suggestion
examples-demo/**
!examples-demo/README.md
```

---

i think it works bc already under version control, but yeah that's clearer